activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*  Add `action_controller_api` and `action_controller_base` hooks to be called in `ActiveSupport.on_load`
```

Comment:
Can we change this to the below?

Add `action_controller_api` and `action_controller_base` load hooks